lib/connection_string.js

```diff
@@ -37,6 +37,10 @@ function matchesParentDomain(srvAddress, parentDomain) {
 function parseSrvConnectionString(uri, options, callback) {
   const result = URL.parse(uri, true);
 
+  if (options.directConnection || options.directconnection) {
```

Comment:
is it possible to have both forms here? I was hoping we wouldn't be introducing more cases where we had to check the upper and lowercase version of URI options.

Reply:
During testing it came in as both, I think it performs differently when in URL and passed in, this covers all our bases.

<img width="624" alt="Screen Shot 2020-05-11 at 3 52 25 PM" src="https://user-images.githubusercontent.com/296798/81605509-8cd3df00-939f-11ea-9b06-7c693f4b1319.png">

Is there a helper I'm missing? It's also in `CASE_TRANSLATION`. 😫

Reply:
so case translation comes into play in [applyConnectionStringOption](https://github.com/mongodb/node-mongodb-native/blob/master/lib/connection_string.js#L299) which is in turn called by [parseQueryString](https://github.com/mongodb/node-mongodb-native/blob/master/lib/connection_string.js#L425). Your change here moves the call to `parseSrvConnectionString` to _after_ `parseQueryString` is called, which means that we should be in a world where all known values are camelCased.

Reply:
@reggi I think we're waiting on you here